### ISSUE TITLE
Limit websites

### ISSUE THREAD

**Issue by SephLeague** [flyth/ts3soundbot#189]
Allow us to choose what websites are enabled and which are not for the commands. Etc, not allowing people to play porn websites (youttube dl supports it) cause some vpses get shut down for that :+1:

**Comment by nrauschcom** [comment]
I would not allow that people to usw the bot in any way. You can also build a wrapper for youtube-dl to do this.

**Comment by flyth** [comment]
Yes, please use a wrapper for now, if you need such functionality. I will try to add such a feature much later nevertheless.
